databricks-dbutils-scala/src/test/scala/com/databricks/sdk/scala/dbutils/integration/SdkDbfsIntegrationTest.scala

```diff
@@ -8,12 +9,11 @@ class SdkDbfsIntegrationTest extends VolumeIntegrationTestBase {
     val testFilePath = s"/tmp/upload_and_download.txt"
     val testFile = "Hello, world!"
 
-    val e = intercept[IllegalArgumentException] {
+    val e = intercept[DatabricksError] {
```

Comment:
I believe this should be a `com.databricks.sdk.core.error.platform.BadRequest`, since it is a 400 with error code `BAD_REQUEST`. If not, I'm curious why this isn't the case.

To be honest, I'm surprised this test ever passed, since I don't see where an IllegalArgumentException was ever thrown by the SDK as a result of an API call.